engine/src/main/java/org/terasology/logic/players/MenuControlSystem.java

```diff
@@ -46,6 +46,7 @@
 
     @Override
     public void initialise() {
+        nuiManager.getHUD().addHUDElement("inventoryGridHUD");
```

Comment:
A comment would be good that explains why we add it here. (e.g. we want it to be behind the toolbar, and the HUD is always in the back). There should also be a TODO to find a better way of doing item drops that do not depend on the visibility of the HUD.